schema/revert/tables/form_001.sql

```diff
@@ -0,0 +1,9 @@
+-- Deploy cif:tables/form to pg
+-- requires: schemas/main
+
+begin;
+
+-- No revert. Cannot change varchar back to regprocedure without error.
+-- Table will be dropped in original revert file.
```

Comment:
I don't completely understand this comment. Sqitch will go all the way back to the original revert file since there isn't a 001 revert?

Reply:
Sqitch runs all revert files in the reverse order of the plan, as sql scripts. If we don't have any code in there, no actions will be run, and sqitch will proceed to the next revert file. Eventually, the revert file for `form.sql` will be hit and the table dropped.